pywinauto/__init__.py

```diff
@@ -104,24 +104,39 @@ def _get_com_threading_mode(module_sys):
 
 
     class Desktop(object):
-
         """Simple class to call something like ``Desktop().WindowName.ControlName.method()``"""
 
-        def __init__(self, backend=None):
+        def __init__(self, backend="win32"):

```

Comment:
I think user must understand explicitly which backend is used. I have an idea to make this argument mandatory for `Application` object as well. But not right now. Maybe in 0.7.0 where we plan to break some compatibility.

Reply:
I will return `None` for backend default value and will add a check for `None` value.

Reply:
Something like this:
```python
if not backend:
    raise ValueError('You should set backend, win32 or uia.')
```